formats/specialist_document/frontend/examples/aaib-reports.json

```diff
@@ -10,6 +10,7 @@
 			],
 			"report_type" : "correspondence-investigation",
 			"date_of_occurrence" : "2015-08-08",
+			"bulk_published" : false,
```

Comment:
Ah yeah, good call. But it does need to be in the `metadata` section for all the formats.